modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/SyncSettingsTask.kt

```diff
@@ -30,15 +30,6 @@ class SyncSettingsTask(val context: Context, val parameters: WorkerParameters) :
                 return Result.failure()
             }
 
-            listOf(
-                settings.freeGiftAcknowledged,
-                settings.marketingOptIn,
-                settings.podcastsSortType,
-                settings.skipBackInSecs,
-                settings.skipForwardInSecs,
-            ).forEach {
-                it.doesNotNeedSync()
-            }
```

Comment:
Getting rid of the need to do this feels like a big win because it's only a matter of time until someone added a new setting that needed to be synced and forgot to include it here (which would have meant the app would have sent values that had just been synced from the server back to the server on the next sync call).

Reply:
Yes, that's super nice!